ethereum/contracts/TestERC20.sol

```diff
@@ -38,4 +38,13 @@ contract TestERC20 is ERC20 {
     function approveInternal(address owner, address spender, uint256 value) public {
         _approve(owner, spender, value);
     }
+
+    // You can send a zero-value transfer directly to the contract address to
+    // get a 100 standard unit tokens.
```

Comment:
I needed a contract address that unit tests could transfer ETH to. Copying this idea from the Weenus token project seemed like the easiest way to enable that capability.